corehq/apps/domain/templates/domain/partials/bootstrap3/payment_modal.html

```diff
@@ -44,6 +45,42 @@
       </div>
     </div>
     <input type="hidden" name="amount" data-bind="value: amount" />
+    <!-- /ko -->
+
+    <!-- ko if: $root.paymentMethod() === $root.WIRE -->
+    <div data-bind="with: invoice_credit">
+      <div class="form-group">
+        <label class="control-label col-sm-2">
+          <span data-bind="text: name"></span>
+        </label>
+        <div class="col-sm-10">
+          <div class="input-group">
+            <span class="input-group-addon">$</span>
+            <input type="text"
```

Comment:
is this HTML new or copied? it seems to not be following the html indentation guidelines

Reply:
https://github.com/dimagi/commcare-hq/pull/36064/commits/7eeaac553e640af2ee6233a12e5cf8e5b93fafb2